Log message:
fix the survivors of fbcon_vbl_handler() renaming

Signed-off-by: Al Viro <viro@zeniv.linux.org.uk>

Modified files:
    linux/drivers/video/console:
	fbcon.c 


Index: linux/drivers/video/console/fbcon.c
Stats: 4 modifications
http://linux-m68k-cvs.ubb.ca/c/cvsweb/linux/drivers/video/console/fbcon%2ec.diff?r1=1.1.1.23&r2=1.4
========================================================================
--- linux/drivers/video/console/fbcon.c	23 Sep 2006 17:01:53 -0000	1.1.1.23
+++ linux/drivers/video/console/fbcon.c	27 Sep 2006 23:15:51 -0000	1.4
@@ -3197,11 +3197,11 @@
 		return;
 
 #ifdef CONFIG_ATARI
-	free_irq(IRQ_AUTO_4, fbcon_vbl_handler);
+	free_irq(IRQ_AUTO_4, fb_vbl_handler);
 #endif
 #ifdef CONFIG_MAC
 	if (MACH_IS_MAC && vbl_detected)
-		free_irq(IRQ_MAC_VBL, fbcon_vbl_handler);
+		free_irq(IRQ_MAC_VBL, fb_vbl_handler);
 #endif
 
 	kfree((void *)softback_buf);
-
To unsubscribe from this list: send the line "unsubscribe linux-m68k-cvscommit" in
the body of a message to majordomo@vger.kernel.org
More majordomo info at  http://vger.kernel.org/majordomo-info.html

